src/docs/codelabs/extended-flex-widget.md

```diff
@@ -0,0 +1,597 @@
+---
+title: "Basic Flutter layout concepts"
+description: "A codelab that teaches basic Flutter layout concepts through DartPad2 code examples."
+toc: true
+--- 
+{{site.alert.important}}
+  This codelab covers basic Flutter layout concepts using a code editor called DartPad. 
+  DartPad is in an experimental stage and hasn't been fully tested on all browsers. If you encounter any difficulties while using DartPad on a specific browser, 
+  please create a [DartPad issue](https://github.com/dart-lang/dart-pad/issues/new) and specify which browser you're using in the issue title.
+{{site.alert.end}}
```

Comment:
If you keep this alert, it should go after the welcome paragraph. No one wants to see a big yellow box first thing on the page!

(I was going to say "remove this box", but since DartPad support for Flutter is new and uses different code paths, maybe this box is warranted.)

Reply:
Yeah, unfortunately the box will remain until Safari supports Flutter. Andrew, Martin, and Zooey had a hand in creating this box, and it seems Jon and Anjan are being called upon for input.

Reply:
Or shall I say, until Safari supports DartPad.